webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js

```diff
@@ -0,0 +1,224 @@
+import {
+  selectCVFilterModuleStream,
+  selectCVFilterModuleStreamStatus,
+  selectCVFilterModuleStreamError,
+  selectCVFilters, selectCVFilterDetails,
+} from '../ContentViewDetailSelectors';
+import getContentViewDetails, {
+  addCVFilterRule, removeCVFilterRule, getCVFilterModuleStreams,
+  deleteContentViewFilterRules, addContentViewFilterRules,
+} from '../ContentViewDetailActions';
+import AddedStatusLabel from '../../../../components/AddedStatusLabel';
+import AffectedRepositoryTable from './AffectedRepositories/AffectedRepositoryTable';
+
+const CVModuleStreamFilterContent = ({
+  cvId, filterId, showAffectedRepos, setShowAffectedRepos,
+}) => {
+  const dispatch = useDispatch();
+  const { results: filterResults } =
+    useSelector(state => selectCVFilters(state, cvId), shallowEqual);
+  const response = useSelector(state =>
+    selectCVFilterModuleStream(state, cvId, filterId), shallowEqual);
+  const status = useSelector(state =>
+    selectCVFilterModuleStreamStatus(state, cvId, filterId), shallowEqual);
+  const error = useSelector(state =>
+    selectCVFilterModuleStreamError(state, cvId, filterId), shallowEqual);
+  const filterDetails = useSelector(state =>
+    selectCVFilterDetails(state, cvId, filterId), shallowEqual);
+  const { repositories = [] } = filterDetails;
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [activeTabKey, setActiveTabKey] = useState(0);
+  const loading = status === STATUS.PENDING;
+  const [bulkActionOpen, setBulkActionOpen] = useState(false);
+  const deselectAll = () => setRows(rows.map(row => ({ ...row, selected: false })));
+  const toggleBulkAction = () => setBulkActionOpen(prevState => !prevState);
+  const hasAddedSelected = rows.some(({ selected, added }) => selected && added);
+  const hasNotAddedSelected = rows.some(({ selected, added }) => selected && !added);
+  const columnHeaders = [
+    __('Name'),
+    __('Stream'),
+    __('Version'),
+    __('Context'),
+    __('Status'),
+  ];
+
+  const buildRows = useCallback((results) => {
+    const newRows = [];
+    const filterRules = filterResults.find(({ id }) => id === Number(filterId))?.rules || [];
+    results.forEach((moduleStreams) => {
+      const {
+        id,
+        name,
+        stream,
+        version,
+        context,
+        filter_ids: filterIds,
+        ...rest
+      } = moduleStreams;
+
+      const added = filterIds.includes(parseInt(filterId, 10));
+
+      const cells = [
+        { title: name },
+        { title: stream },
+        { title: version },
+        { title: context },
+        { title: <AddedStatusLabel added={added} /> },
+      ];
+
+      /* eslint-disable camelcase */
+
+      newRows.push({
+        cells,
+        module_stream_id: id,
+        packagefilterId: filterRules?.find(({ id }) => id === id)?.id, // eslint-disable-line max-len, no-shadow, no-self-compare
```

Comment:
This needs to look for module_stream_id equal to module_strem.id..
Basically this:
`filterRules?.find(({ module_stream_id }) => module_stream_id === id)?.id`

Reply:
also, could you rename the variable to something like moduleStreamRuleId..